calendar_v17/odoo_office365_calendar/models/office365lib.py

```
@@ -0,0 +1,44 @@
+        vals = json.dumps(vals)
+        conn.request(request_type, "/v1.0/me/" + request, vals, headers={
+            'Authorization': self.Session.token,
+            'Content-Type': 'application/json'
+        })
+        if request_type == 'DELETE':
+            return conn.getresponse()
+        response = simplejson.loads(conn.getresponse().read())
+        return json.dumps(response)
+
+
+class Office365:
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)